src/ast.js

```diff
@@ -219,6 +219,46 @@ export class FunctionDefinition extends ASTNode {
   }
 }
 
+export class CondClause extends ASTNode {
+  constructor(from, to, testExpr, thenExprs, options={}) {
+    super(from, to, 'condClause', options);
+    this.testExpr = testExpr;
+    this.thenExprs = thenExprs;
```

Comment:
Looking at https://docs.racket-lang.org/reference/if.html it seems like multiple then expressions are allowed in racket:

> 
> [test-expr then-body ...+]
> The then-bodys are evaluated in order, and the results from all but the last then-body are ignored. The results of the last then-body, which is in tail position with respect to the cond form, provides the result for the whole cond form.

Wescheme follows the racket spec right?